### PR TITLE
V1.0

### PR DESCRIPTION
**Pull request by VSlobolinskyi**
Development branch

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export default {
+	name: "app",
+	components: {
+		building,
+		about
+	}
+}
+</script>
+
+<style scoped>
```
Comment (leleleadele) [inline src/App.vue]: if the tags are empty, you can delete them

File: src/assets/icons/cart_icon.svg [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
```
Comment (leleleadele) [inline src/assets/icons/cart_icon.svg]: weird file

File: src/assets/icons/facebook_icon.svg [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
```
Comment (leleleadele) [inline src/assets/icons/facebook_icon.svg]: another weird file :D many useless grouptags

File: src/components/molecules/about.vue [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<template>
+	<div class="about">
+		<aboutTitle/>
+		<aboutSlogan/>
```
Comment (leleleadele) [inline src/components/molecules/about.vue:4]: I think common practice is to either write component tags LikeThis or like-this, not in camelCase. please check this issue

**Review by leleleadele — APPROVED** [review]

**Review by ArtemsBoroviks — APPROVED** [review]

File: public/index.html [hunk 5]
```diff
@@ -3,9 +3,11 @@
 	<head>
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="src/styles/globalStyles.css">
 		<title>renome_vue</title>
 	</head>
 	<body id="body">
 		<div id="app"></div>
 	</body>
+	<style></style>
```
Comment (Laacmeele) [inline public/index.html:12]: Empty style tags seem unnecessary

**Pull request merged**